packages/config/src/config/constants.ts

```diff
@@ -34,17 +34,9 @@ export const CERT_PUBLIC_KEY_PATH =
   '../../.secrets/public-key.pem'
 export const PRODUCTION = process.env.NODE_ENV === 'production'
 export const QA_ENV = process.env.QA_ENV || false
+export const FHIR_URL = process.env.FHIR_URL || 'http://localhost:3447/fhir/'
```

Comment:
Is there a country config PR where this is added?

Reply:
Added it to docker-composes in core now, but I couldn't find the place where to add it in country config. The only place which configures this in country config is https://github.com/opencrvs/opencrvs-countryconfig/blob/a529f42455ea52526fa8d21a8b21994f422fe79a/infrastructure/docker-compose.staging-deploy.yml#L103 the countryconfig container itself.